### PR TITLE
Startswith and icontains in polymorphic API

### PR DESCRIPTION
**Pull request by szok**
Startswith and icontains in polymorphic API

### CODE REVIEW

**Comment by ghost** [comment]
:ok:

**Comment by mkurek** [comment]
we need unit tests for that :)

File: src/ralph/api/viewsets.py [hunk 1]
```diff
@@ -124,6 +124,7 @@ class RalphAPIViewSet(
     metaclass=RalphAPIViewSetMetaclass
 ):
     extend_filter_fields = None
+    polymorphic_limit = 50
```
Comment (mkurek) [inline src/ralph/api/viewsets.py]: why do we need limit here? isn't it handled by api pagination?

**Comment by mkurek** [comment]
:ok:

**Pull request merged**